<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-Q37MKND" name="com.bitnei.apitest.testcases.openapi.RunningData" tests="4" failures="1" timestamp="02 六月 2020 07:56:41 GMT" time="10.250" errors="0">
  <testcase name="EnergyConservationInvalidtoken" time="0.049" classname="com.bitnei.apitest.testcases.openapi.RunningData"/>
  <testcase name="EnergyConservationNormal" time="10.085" classname="com.bitnei.apitest.testcases.openapi.RunningData">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;status&quot;:{&quot;currentValue is &quot;:400,&quot;expectedValue is&quot;:200},&quot;error&quot;:{&quot;currentValue is &quot;:true,&quot;expectedValue is&quot;:false}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"status":{"currentValue is ":400,"expectedValue is":200},"error":{"currentValue is ":true,"expectedValue is":false}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.openapi.RunningData.EnergyConservationNormal(RunningData.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- EnergyConservationNormal -->
  <testcase name="EnergyConservationUpScope" time="0.062" classname="com.bitnei.apitest.testcases.openapi.RunningData"/>
  <testcase name="EnergyConservationNolicensePlate" time="0.054" classname="com.bitnei.apitest.testcases.openapi.RunningData"/>
</testsuite> <!-- com.bitnei.apitest.testcases.openapi.RunningData -->
